### ISSUE TITLE
Site should be fully functional with only keyboard

### ISSUE THREAD

**Issue by sdellis** [pulibrary/dpul-collections#670]
### Summary or User Story
The site should be fully functional using only the keyboard to navigate. Currently, many buttons, such as _View_ and _Metadata_ on the Item page, need clear focus indicators for users who use the keyboard to navigate websites. Browse Cards and Search Rows (or links) don't seem to be tab-able either. 

### Acceptance Criteria

This section can be completed by the DLS team. Some requirements to consider
including:

- [ ] All pages are usable with only a keyboard.

### First step

Try to complete basic use cases with only a keyboard and make note of blockers.
